tests/curious_tests/test_simple_queries.py

```diff
@@ -76,7 +76,26 @@ def test_query_traversing_to_fk_object(self):
     self.assertEquals(result[0][0][1], -1)
     assertQueryResultsEqual(self, result[0][0][0], [(self.blogs[0], None)])
     self.assertEquals(result[1], Blog)
-  
+
+  def test_query_traversing_to_one_to_one_object(self):
+    user_test = UserTest(name='user a')
+    user_test.save()
+    user_one_to_one = UserOneToOne(user=user_test, extra='extra')
+    user_one_to_one.save()
+
+    qs = 'UserOneToOne(extra="extra") UserOneToOne.user'
+    query = Query(qs)
+    result = query()
+    assertQueryResultsEqual(self, result[0][0][0], [(user_test, None)])
+    self.assertEquals(result[1], UserTest)
+
+    # go back
```

Comment:
passing in self to an unbound method is ticky here. Csn this method be added via a mixin instead?

Reply:
It's a copy paste from how other test methods are done.